docs/README.md

```diff
@@ -68,14 +68,73 @@ macOS (OS X) development environment.
-* [Permission management](web/permissions.md)
-* [Usage of PostgreSQL database](web/postgresql_setup.md)
-* [How to deal with false positives](analyzer/false_positives.md)
-* [Store Cppcheck analyzer reports](/tools/report-converter/README.md#cppcheck)
+* [Avoiding or suppressing false positives](analyzer/false_positives.md)
+* [Checker and Static Analyzer configuration](analyzer/checker_and_analyzer_configuration.md)
+* [GCC incompatibilities](analyzer/gcc_incompatibilities.md)
+* [Suppressing false positives](analyzer/user_guide.md#source-code-comments)
+
+## Web based report management
+* [Webserver User Guide](web/user_guide.md)
+* [Command line and WEB UI Feature overview](feature_comparison.md)
+
+ Security configuration 
+* [Configuring Authentication](web/authentication.md)
+* [Configuring Authorization](web/permissions.md)
+
+Deployment
+* [Deploy server using docker](web/docker.md#deployment)
+
+Server Configuration
+* [Configuring Server Logging](logging.md)
+* [Setting up multiple CodeChecker repositories in one server](web/products.md)
+
+Continous Integration(CI)
+* [Setting up CI gating with Gerrit and Jenkins](jenkins_gerrit_integration.md)
+
+Database Configuration
+* [PostgreSQL database backend setup guide](web/PostgreSQL_setup.md)
+* [CodeChecker server and database schema upgrade guide](web/db_schema_guide.md)
+
+
+### Storage of reports from analyzer tools
+CodeChecker can be used as a generic tool for visualizing analyzer results.
+
+The following tools are supported:
+
+| Language       | Analyzer     |
+| -------------- |--------------|
+| **C/C++**      | [Clang Static Analyzer](https://clang-analyzer.llvm.org/)    |
+|                | [Clang Tidy](https://clang.llvm.org/extra/clang-tidy/)  |
+|                | [Clang Sanitizers](supported_code_analyzers.md#clang-sanitizers)    |
+|                | [Cppcheck](/tools/report-converter/README.md#cppcheck)    |
+|                | [Facebook Infer](/tools/report-converter/README.md#fbinfer)    |
+| **Java**       | [SpotBugs](/tools/report-converter/README.md#spotbugs)    |
+|                | [Facebook Infer](/tools/report-converter/README.md#fbinfer)    |
+| **Python**     | [Pylint](/tools/report-converter/README.md#pylint)    |
+|                | [Pyflakes](/tools/report-converter/README.md#pyflakes)    |
+| **JavaScript** | [ESLint](/tools/report-converter/README.md#eslint)    |
+| **TypeScript** | [TSLint](/tools/report-converter/README.md#tslint)    |
+| **Go**         | [Golint](/tools/report-converter/README.md#golint)    |
+
+
+For details see 
+[supported code analyzers](supported_code_analyzers.md) documentation and the 
+[Report Converter Tool](/tools/report-converter/README.md).
+
+## Common Tools
+Useful tools that can also be used outside CodeChecker.
+
+* [Build Logger (to generate JSON Compilation Database from your builds)](analyzer/tools/build-logger) 
+* [Plist to HTML converter](/tools/plist_to_html/README.md)
```

Comment:
For other tools we give a short description what the tool does. Maybe its obvious what plist-to-html does but it would be great to add a short description for this tool too:
```suggestion
* [Plist to HTML converter (to generate HTML files from the given plist files)](/tools/plist_to_html/README.md)
```